src/loaders/workspace_downloader/workspace_downloader.py

```diff
@@ -275,8 +323,19 @@ def main():
     if args.workers < 1 or args.workers > cpu_count():
         parser.error(f"minimum worker is 1 and maximum worker is {cpu_count()}")
 
-    (workspace_id, root_dir, kb_base_url, workers, token_filepath, keep_job_dir) = (
+    (
+        workspace_id,
+        collection,
+        source_version,
+        root_dir,
+        kb_base_url,
+        workers,
+        token_filepath,
+        keep_job_dir,
+    ) = (
```

Comment:
It's be easier to read and less lines if it was just
```
workspace_id = args.workspace_id
collection = args.collection
...
```

Reply:
Updated

Reply:
:+1: